powerline_shell/__init__.py

```diff
@@ -64,6 +64,7 @@ class Powerline(object):
 
     color_templates = {
         'bash': r'\[\e%s\]',
+        'fish': '%s',
```

Comment:
There's no need for this additional shell type. The `bare` shell type is meant to represent fish. It's not my favorite thing that it works this way, but that's what is going on here.